gcc/rust/error_codes.incl

```diff
@@ -0,0 +1,529 @@
+// Copyright (C) 2020-2023 Free Software Foundation, Inc.
+
+// This file is part of GCC.
+
+// GCC is free software; you can redistribute it and/or modify it under
+// the terms of the GNU General Public License as published by the Free
+// Software Foundation; either version 3, or (at your option) any later
+// version.
+
+// GCC is distributed in the hope that it will be useful, but WITHOUT ANY
+// WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+// for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with GCC; see the file COPYING3.  If not see
+// <http://www.gnu.org/licenses/>.
+// This file was used by rust-diagnostic.h to generate the error codes
+// clang-format off
+
+// https://gist.github.com/MahadMuhammad/8c9d5fc88ea18d8c520937a8071d4185
+#ifdef RUST_ERROR_CODES_H
+ERROR(E0001),// this error code is no longer emitted by the compiler
+ERROR(E0002),// this error code is no longer emitted by the compiler
```

Comment:
Is this location suited for this file, or should I move it to util folder like `util/error_codes.incl`?